stable/jenkins/templates/_helpers.tpl

```diff
@@ -153,7 +164,11 @@ Returns kubernetes pod template configuration as code
     envVars:
     - containerEnvVar:
         key: "JENKINS_URL"
+    {{- if .Values.master.slaveJenkinsUrl }}
+        value: {{ .Values.master.slaveJenkinsUrl }}
```

Comment:
to allow templating this will be `value: {{ tpl .Values.master.slaveJenkinsUrl . }}`